app/assets/stylesheets/static_pages_home.scss

```diff
@@ -0,0 +1,41 @@
+.static_pages_home {
+	height: 100vh; width: 100vw;
+	background: #F0F0F0;
+
+	.row {
+		position: relative; top: 45%;
+		transform: translateY(-50%);
+		h2 {
+			font-family: 'Arvo', sans-serif;
+			color: #ff8d7a; text-align: center;
+			font-size: 5em;
```

Comment:
Are you sure you want this in em? Plz check on mobile phone.
In my experience, vmin works better than em for such huge fonts.